src/upload_java_tools.sh

```diff
@@ -97,7 +97,7 @@ To build from source the same zip run the commands:
 
 $ git clone https://github.com/bazelbuild/bazel.git
 $ git checkout ${commit_hash}
-$ bazel build //src:java_tools_prebuilt.zip
+$ bazel build //src:java_tools.zip
```

Comment:
I think the same script is used for both prebuilt and non-prebuilt zips. This change would fix one but break 3 other platform. Please use `${platform}` variable here.